sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -168,11 +168,13 @@ def input_elements(self, instruction_id, expected_targets):
 
   def output_stream(self, instruction_id, target):
     def add_to_send_queue(data):
```

Comment:
Add a TODO about chunking the output instead of buffering all the data before emitting any of it.